Tests/Foundation/Tests/TestFileManager.swift

```diff
@@ -410,6 +410,9 @@ class TestFileManager : XCTestCase {
         
         do {
             try fm.setAttributes([.posixPermissions : NSNumber(value: Int16(0o0600))], ofItemAtPath: path)
+#if !os(Windows)
+            try fm.setAttributes([.modificationDate: NSDate(timeIntervalSince1970: 1234567890.5)], ofItemAtPath: path)
```

Comment:
Nitpick: could you move `1234567890.5` value to a `let` binding to reuse it in `XCTAssertEqual` below?